src/components/Main/Main.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useState, useMemo } from 'react';
+
+import './Main.scss';
+import { TodoList } from '../TodoList';
+import { DispatchContext, FilterContext, StateContext } from '../TodosContext';
+import { Status } from '../../types/Status';
+import { Todo } from '../../types/Todo';
+
+export const Main: React.FC = () => {
+  const todos = useContext(StateContext);
+  const dispatch = useContext(DispatchContext);
+  const filterStatus = useContext(FilterContext);
+
+  const [checked, setChecked] = useState(false);
```

Comment:
Rename boolean variable according [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig). Fix all cases if present
```suggestion
  const [isChecked, setIsChecked] = useState(false);
```